explanations/machine-learning-terms.md

```diff
@@ -25,6 +25,22 @@ Researchers often invent complicated names for simple things. This page will hel
     <p>One example is working with words in Natural Language Processing. Words are usually transformed to a list of numbers  (embedded) before given as input to a neural network. The exact mapping between word and the list of numbers will be automatically learned.</p>
 </td>
 </tr>
+<tr>
+<td valign="top">Principal Component Analysis (PCA)</td>
+<td>
+    <p>An algorithm that is used to narrow down datasets.</p>
+    <p>PCA (as any 'dimensionality reduction' algorithms) has several advantages:
+	<ul>
+		<li>it reduces the storage space required</li>
+		<li>it improves the performance of the machine learning model by removing the multicollinearity in the data</li>
+		<li>it eases the visualization of the data when reduced to very low dimensions such as 2D or 3D</li>
+	</ul>
```

Comment:
I have the feeling this is too much information. Maybe you can summarize it more?

Reply:
What do you think of this:
"PCA (as any 'dimensionality reduction' algorithms) helps to reduce the storage space, to improve the performance of the models, and, to ease the data visualization."

Reply:
Yes, this is good 👍🏻